lib/std/Build.zig

```diff
@@ -1812,14 +1913,22 @@ fn pathFromCwd(b: *Build, sub_path: []const u8) []u8 {
     return b.pathResolve(&.{ cwd, sub_path });
 }
 
+/// Naively combines a series of paths with the native path separator.
+/// Allocates memory for the result, which must be freed by the caller.
+/// See: `std.fs.path.join`
```

Comment:
This seems mostly redundant with what I perceive to be intuitive information derived from the function name and signature.